### DISCUSSION TITLE
New Pricing Plan for Outline Selfhosted

### DISCUSSION THREAD

**Discussion by sats268842** [outline/outline#6802] [Self Hosting]
I want to let you that we need a price strategy for outline selfhosted a one-time payment that provides some lifetime additional capabilities. For example, see https://plane.so/one.
@tommoor

**Comment by Pratkp2022** [comment]
I'm also looking for this new pricing strategy.

**Comment by electric-m** [comment]
I sthat meant for the community version as well (private user),or for the on-premise business self hoster?

**Reply by sats268842** [reply]
It will be helpful for on-premise business self-hosters
